hack/timestamps.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+function timestamps::time_wrapper() {
+    while IFS= read -r line; do
+        printf "%(%T)T: %s\n" "-1" "$line"
+    done
+}
+
+if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
+    /bin/sh "$@" 2>&1 | timestamps::time_wrapper
```

Comment:
`/bin/sh "$@" > >(timestamps::time_wrapper) 2> >(timestamps::time_wrapper)` would avoid hiding the return value and doesn't merge stdout and stderr.
Nit: why the weird function name?

Reply:
Nice, I didn't think think of this way. I'll update my fix.

About the function name, it ensures that if someone sources this script to use this function, the context from where this function came from remains.